package.json

```diff
@@ -18,7 +18,7 @@
     "dev-proofs": "npm run build:webpack && ts-node ./tests/browserTests/proofs-server.ts",
     "prepublishOnly": "npm run test && npm run build",
     "lint": "eslint --ext .ts ./src",
-    "build-documentation": "shx rm -rf docs && typedoc --tsconfig tsconfig.typedoc.json --out docs src && typedoc --tsconfig tsconfig.typedoc.json --json docs/docs.json src",
+    "build-documentation": "shx rm -rf docs && cross-env typedoc --name \" blockstack.js $npm_package_version Library Reference \" --tsconfig tsconfig.typedoc.json --out docs src && cross-env typedoc --name \" blockstack.js $npm_package_version Library Reference \" --tsconfig tsconfig.typedoc.json --json docs/docs.json src",
```

Comment:
Are the spaces between the name quotes intentional here?
![image](https://user-images.githubusercontent.com/1447546/56679372-ea14c980-6692-11e9-8908-b30c2eb9ba0b.png)
![image](https://user-images.githubusercontent.com/1447546/56679395-fa2ca900-6692-11e9-997d-89a7d449c61e.png)

Reply:
Just something i was doing during testing.